torch/distributed/_tensor/ops/math_ops.py

```diff
@@ -350,6 +351,164 @@ def layer_norm_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> OpStrategy:
     return output_strategy
 
 
+@register_op_strategy(
+    [aten.native_layer_norm_backward.default],
```

Comment:
Thanks for adding the backward support. I think we also need to remove the existing `aten.native_layer_norm_backward.default` from experimental_op.py.

Reply:
@yoyoyocmu you're right. Regarding correctness, the rule in this PR has higher priority than the experimental one (the experimental rule is shadowed). I will remove the experimental op in #115689